logging.c

```diff
@@ -180,13 +226,108 @@ static void owl_log_entry_free(void *data)
   }
 }
 
+#if GLIB_CHECK_VERSION(2, 32, 0)
```

Comment:
I've reported a [ticket on trac](https://barnowl.mit.edu/ticket/293#ticket) to note that we should do this in a follow-up PR (and in more places than just the two in this file).